src/main/java/com/faforever/client/vault/search/SearchController.java

```diff
@@ -428,10 +429,6 @@ public void setRootType(Class<?> rootType) {
     initialLogicalNodeController.specificationController.setRootType(rootType);
   }
 
-  public void setSearchButtonDisabledCondition(BooleanBinding inSearchableState) {
-    searchButton.disableProperty().bind(queryTextField.textProperty().isEmpty().or(inSearchableState.not()));
-  }
-
```

Comment:
just remove the queryTextField.textProperty().isEmpty() and only bind on inSearchableState.not()